assets/js/translator.js

```diff
@@ -58,6 +83,15 @@ if(modeEnabled('translation')) {
             persistChoices('translator', true);
         });
 
+        $('input.chainedTranslation').change(function () {
```

Comment:
Since it's only used once, I'd suggest using an id here and in other places, i.e. `#chainedTranslation`. Classes are more semantically meaningful when there are multiple elements with it.